### PR TITLE
CORE-5232 added new system value for tenant id

### PR DESCRIPTION
**Pull request by JMCullimoreDell**

### CODE REVIEW

File: ManyWhoConstants.cs [hunk 1]
```diff
@@ -373,6 +373,9 @@ public class ManyWhoConstants
         public static readonly Guid MANYWHO_FALSE_VALUE_ID = Guid.Parse("496FD041-D91F-48FB-AA4F-91C6C9A11CA1");
         public const string MANYWHO_FALSE_DEVELOPER_NAME = "$False";
 
+        public static readonly Guid MANYWHO_TENANT_ID_VALUE_ID = Guid.Parse("247AEE82-B65F-43AF-B28D-29E2458EF1E4");
+        public const string MANYWHO_TENANT_ID_DEVELOPER_NAME = "$TenantId";
```
Comment (jonjomckay) [inline ManyWhoConstants.cs]: Once this is placed under the `$State` object, it should be named `Tenant ID` as there's no reason we need to mangle or change the capitalisation of names here.

**Review by jonjomckay — CHANGES_REQUESTED** [review]

File: ManyWhoConstants.cs (hunk 1 above)
Comment (JMCullimoreDell) [reply ManyWhoConstants.cs]: Sure, once it's under $State we won't need this any more either, so I'll cancel this PR

**Comment by JMCullimoreDell** [comment]
Moving to $State system variable

**Comment by JMCullimoreDell** [comment]
Just noticed that we will still need this but it does need updating

File: ManyWhoConstants.cs (hunk 1 above)
Comment (JMCullimoreDell) [reply ManyWhoConstants.cs]: Moved to `$State -> Tenant ID`

**Review by jonjomckay — APPROVED** [review]

**Pull request merged**